config/deprecation-workflow.js

```diff
@@ -18,5 +17,9 @@ self.deprecationWorkflow.config = {
         { handler: 'silence', matchId: 'ember-engines.deprecation-camelized-engine-names' },
         { handler: 'silence', matchId: 'ember-data:legacy-test-helper-support' },
         { handler: 'silence', matchId: 'has-block-and-has-block-params' },
+        { handler: 'silence', matchId: 'ember-simple-auth.initializer.setup-session-restoration' },
+        { handler: 'silence', matchId: 'ember-simple-auth.events.session-service' },
+        { handler: 'silence', matchId: 'ember-cli-mirage.miragejs.import' },
+        { handler: 'silence', matchId: 'ember-cli-mirage-config-routes-only-export' },
```

Comment:
So we un-silence one deprecation warning and add 4 more rules to silence deprecation warnings? Is it because removing the `ember-metal.get-with-default` warning requires upgrading `ember-simple-auth`?

Reply:
Yeah, and mirage had to be upgraded as well.